src/GameLogic/InventoryStorage.cs

```diff
@@ -124,7 +124,7 @@ public override async ValueTask RemoveItemAsync(Item item)
         await base.RemoveItemAsync(item).ConfigureAwait(false);
         if (isEquippedItem && this.EquippedItemsChanged is { } eventHandler)
         {
-            await eventHandler(new ItemEventArgs(item)).ConfigureAwait(false);
+            await eventHandler(new ItemEventArgs(item, false)).ConfigureAwait(false);
```

Comment:
equipped->unequipped: notify that isEquipped is false (ItemGroup byte will be marked as 0xFF)